.github/CODEOWNERS

```diff
@@ -22,3 +22,4 @@
 /traffic_director/      @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
 /vpc/                   @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
 /managedkafka/          @terraform-google-modules/managedkafka-dev-team @terraform-google-modules/terraform-samples-reviewers
+/backupdr/              @terraform-google-modules/gcbdr-samples-team @terraform-google-modules/terraform-samples-reviewers
```

Comment:
The folder `backupdr` doesn't match the code you added `gcbdr`. Please rename the added code to `backupdr` (to also match the product code that's used in your region tags)

Reply:
Updating to gcbdr everywhere, as it would be more appropriate

Reply:
> I would also suggest you match your team name with the product name

@glasnt this was already provisioned and the team mdb group was added. If not a hard requirement, would prefer to keep `gcbdr-samples-team` to reduce churn.